packages/story-content-format/src/index.ts

```diff
@@ -343,3 +354,7 @@ function validateRecursiveListNode(value: any, type?: ListNode.Type): RecursiveL
         ? Core.ListNode.validateListNode(value, type, validateBlock)
         : Core.ListNode.validateListNode(value, validateBlock);
 }
+
+export import TableNode = Core.TableNode;
```

Comment:
Just a note that these changes should be also applied to the `email-content-format` definitions. As we do allow tables inside emails as well.